qe_gluster_firewall.yml

```diff
@@ -0,0 +1,16 @@
+---
+# HACK
+# FIXME: There should be proper firewall configuration!
+
+- hosts: gluster:usm_server
+  remote_user: root
+  tasks:
+  - name: stop firewall
+    service:
+      name: firewalld
+      state: stopped
+      enabled: no
+    ignore_errors: yes
+  - name: pause for 10 second
```

Comment:
Could you add a comment with quick explanation of this task, eg. why is this needed?